schema.sql

```diff
@@ -3,7 +3,7 @@ CREATE TABLE books(
   id SERIAL PRIMARY KEY,
   title VARCHAR(255),
   author VARCHAR(255),
-  image_url VARCHAR(255),
+  image_url TEXT,
```

Comment:
It makes sense to have it longer than 255 - I'd choose a number that was a little more constrained based on the SO post though (2000 maybe?)